src/resolvers.ts

```diff
@@ -2,41 +2,71 @@ import * as facilityService from './services/facilityService'
 import * as healthcareProfessionalService from './services/healthcareProfessionalService'
 import * as gqlType from './typeDefs/gqlTypes'
 import * as submissionService from './services/submissionService'
+import CustomErrors from './errors'
 
 const resolvers = {
     Query: {
         facilities: async () => {
-            const matchingFacilities = await facilityService.searchFacilities(['1'])
+            try {
+                const matchingFacilities = await facilityService.searchFacilities(['1'])
 
-            return matchingFacilities
+                return matchingFacilities
+            } catch (error) {
+                return CustomErrors.notFound(`Failed to find facilities: ${error}`)
+            } 
         },
         facility: async (_parent: gqlType.Facility, args: { id: string; }) => {
-            const matchingFacility = await facilityService.getFacilityById(args.id)
+            try {
+                if (!args.id || !args.id.trim()) {
+                    throw new Error('An ID was not provided')
+                }
+                const matchingFacility = await facilityService.getFacilityById(args.id)
 
-            return matchingFacility
+                return matchingFacility            
+            } catch (error) {
+                return CustomErrors.notFound(`Failed to find the facility: ${error}`)
+            } 
         },
         // healthcareProfessionals: async () => {
         //     const matchingProfessionals = await healthcareProfessional.searchHealthcareProfessionals(['1'])
 
         //     return matchingProfessionals
         // },
         healthcareProfessional: async (_parent: gqlType.HealthcareProfessional, args: { id: string; }) => {
-            const matchingHealthcareProfessional = 
-            await healthcareProfessionalService.getHealthcareProfessionalById(args.id)
+            try {
+                if (!args.id || !args.id.trim()) {
+                    throw new Error('An ID was not provided')
+                }
+                const matchingHealthcareProfessional = 
+                await healthcareProfessionalService.getHealthcareProfessionalById(args.id)
 
-            return matchingHealthcareProfessional
+                return matchingHealthcareProfessional        
+            } catch (error) {
+                return CustomErrors.notFound(`${error}`)
```

Comment:
This error message is less descriptive because the test is written in the way that it should be identical to the test.
I would like and change this with this issue: #262

Reply:
Always update your tests to match your code, not the other way around. Let's fix it in the issue you added ✅